src/pages/routes-and-stops/routes-and-stops.component.ts

```diff
@@ -72,6 +74,19 @@ export class RoutesAndStopsComponent {
       }
     }
   }
+  
+  showToast(message: string): void {
+    if (this.toast) {
+      this.toast.dismissAll();
+    }
+  
+  this.toast = this.toastCtrl.create({
```

Comment:
Trying to use plugins for my vimrc. Probably doing things wrong because typescript files are mostly gray text in my editor. I'll look into this later on today.